src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.csproj

```diff
@@ -89,19 +89,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="build\common\MSTest.TestAdapter.props" Condition=" '$(TargetFramework)' != '$(UwpMinimum)' ">
+    <None Update="build\common\*" Condition=" '$(TargetFramework)' != '$(UwpMinimum)' ">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="build\netfx-netcore-netstandard\MSTest.TestAdapter.targets">
```

Comment:
This is gone completely now? Was it not needed?

Reply:
I don't see the file. Do you?

Reply:
Never mind. It's not there.